reactapp/src/components/UploadImages.js

```diff
@@ -0,0 +1,75 @@
+import React, {Component, createRef} from 'react';
+import Dropzone from 'react-dropzone';
+import "../styles/image_uploader.css";
+import {post} from 'axios';
+
+//Setup for dropzone component. createRef is for creating access/reference to the HTML page's DOM
+const dropzoneRef = createRef();
+//Define a function openDialog, that can open the file picker when you click the button to select files
+//from a folder to upload
+const openDialog = () => {
+    // Note that the ref is set async,
+    // so it might be null at some point
+    if (dropzoneRef.current) {
+        dropzoneRef.current.open()
+    }
+};
+
+export default class UploadImages extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            files: []
+        }
+    }
+    //Method called when a new file is dropped or selected using the upload section
+    //files is an array of File objects  = https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/file
+    onDrop = (files) => {
+        this.setState({
+            files: this.state.files.concat(files),
+        });
+    }
+
+    //Method called upload button is submitted.
+    //e is the input event that is automatically passed for this ui event
+    handleSubmit = (e) => {
+        e.preventDefault() // Stop form submit
+        //Iterate through to submit all files
+        this.props.fileLoader(this.state.files)
+        //upload the first file from the list of smiles(remember state.files is the only you can access the files
+        //from onDrop outside of that method. (example of state functioning like a global variable)
```

Comment:
Let's remove unused commented code. It's a technical debt and the commit history will store it if we need it.